src/xo-app/jobs/overview/index.js

```diff
@@ -0,0 +1,426 @@
+import ActionButton from 'action-button'
+import ActionRowButton from 'action-row-button'
+import ActionToggle from 'action-toggle'
+import Icon from 'icon'
+import React, { Component } from 'react'
+import _, { FormattedDuration } from 'messages'
+import ceil from 'lodash/ceil'
+import classnames from 'classnames'
+import filter from 'lodash/filter'
+import forEach from 'lodash/forEach'
+import map from 'lodash/map'
+import orderBy from 'lodash/orderBy'
+import { FormattedDate } from 'react-intl'
+import { ButtonGroup, Pagination } from 'react-bootstrap-4/lib'
+import { Link } from 'react-router'
+import { confirm } from 'modal'
+import { connectStore } from 'utils'
+import { createGetObject, createPager } from 'selectors'
+
+import {
+  Card,
+  CardHeader,
+  CardBlock
+} from 'card'
+
+import {
+  deleteJobsLog,
+  deleteSchedule,
+  disableSchedule,
+  enableSchedule,
+  runJob,
+  subscribeJobs,
+  subscribeJobsLogs,
+  subscribeSchedules,
+  subscribeScheduleTable
+} from 'xo'
+
+import { getJobValues } from '../helpers'
+
+// ===================================================================
+
+const jobKeyToLabel = {
+  genericTask: _('customJob')
+}
+
+// ===================================================================
+
+const LOGS_PER_PAGE = 10
+
+@connectStore(() => {
+  const object = createGetObject((_, props) => props.value)
+  return (state, props) => ({object: object(state, props)})
+})
+class JobValue extends Component {}
+
+class JobParam extends JobValue {
+  render () {
+    const {
+      object,
+      paramKey,
+      paramValue
+    } = this.props
+    const displayKey = object && object.type || paramKey
+    const displayValue = object && (object.name_label || object.name) || paramValue
+    return <span><strong>{displayKey}:</strong> {displayValue} </span>
+  }
+}
+
+class JobReturn extends JobValue {
+  render () {
+    const {
+      object,
+      returnValue
+    } = this.props
+    let xoName = object && (object.name_label || object.name) && (xoName += object.type && ` (${object.type})` || '')
+    return <span> <Icon icon='arrow-right' /> {xoName || String(returnValue)}</span>
```

Comment:
Extraneous space between `<span>` and `<Icon>` and you may want to insert an explicit space before `xoName`: `{' '}{xoName || String(returnValue)}` because AFAIU your space is *eaten* by JSX.